study-html/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Progate</title>
+    <link rel="stylesheet" href="stylesheet.css">
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+  </head>
+  <body>
+    <header>
+      <div class="container">
+        <div class="header-left">
+          <img class="logo" src="https://prog-8.com/images/html/advanced/main_logo.png">
+        </div>
+        <div class="header-right">
+          <a href="#" class="login">ログイン</a>
+        </div>
+      </div>
+    </header>
+    <div class="top-wrapper">
+      <div class="container">
+        <h1>LEARN TO CODE.</h1>
```

Comment:
html のbodyでは h1 （最上位の見出し）がタイトルとして取り扱われます。そのため、原則 `h1` は1ページ内に一つしか使ってわいけないとなっています。また、今回のページのタイトルはどのコンテンツになると思いますか？
私は `LEARN TO CODE.` や `LEARN TO BE CREATIVE.` ではないと思います。
```suggestion
        <div>LEARN TO CODE.</div>
```